rules/REQUEST-920-PROTOCOL-ENFORCEMENT.conf

```diff
@@ -937,8 +937,9 @@ SecRule &TX:COMBINED_FILE_SIZES "@eq 1" \
 # - text/plain; charset="UTF-8"
 # - multipart/form-data; boundary=----WebKitFormBoundary12345
 # - application/soap+xml; charset=utf-8; action="urn:localhost-hwh#getQuestions"
-#
-SecRule REQUEST_HEADERS:Content-Type "!@rx ^[\w/.+-]+(?:\s?;\s?(?:action|boundary|charset|start(?:-info)?|type|version)\s?=\s?['\"\w.()+,/:=?<>@#-]+)*$" \
+# - application/*+json
+
+SecRule REQUEST_HEADERS:Content-Type "!@rx ^[\w/.+-\*]+(?:\s?;\s?(?:action|boundary|charset|start(?:-info)?|type|version)\s?=\s?['\"\w.()+,/:=?<>@#-\*]+)*$" \
```

Comment:
The `-` in the characterclass should be the last (or the first) character. So the `*` should probably be inserted before the `-`

Reply:
Also, I'm not sure if we need or are allowed to escape the `*` inside the square bracket character class. Probably not. I mean, the `+` is also not escaped.

Reply:
Thank you, will update!